src/index.ts

```diff
@@ -140,17 +162,25 @@ export default class OpentracingExtension<TContext extends SpanContext>
     });
 
     context.addSpan(span, info);
-
     // expose to field
     info.span = span;
 
+    if (this.onFieldResolve) {
+      this.onFieldResolve(source, args, context, info);
+    }
+
     return (error: Error | null, result: any) => {
-      if (error) {
-        span.log({ error: JSON.stringify(error) });
-      }
-      if (result) {
-        span.log({ result: JSON.stringify(result) });
+      if (this.onFieldResolveFinish) {
+        this.onFieldResolveFinish(error, result, span);
+      } else {
+        if (error) {
```

Comment:
Let's remove the default stuff here, this can be done manually thanks to your changes :)

Reply:
thank you for the quick feedback. I just changed it